tests/python-pytest/onnx/backend_rep.py

```diff
@@ -64,9 +64,16 @@ def run(self, inputs, **kwargs):
         else:
             raise NotImplementedError("Only CPU context is supported for now")
 
-        mod = mx.mod.Module(symbol=self.symbol, data_names=['input_0'], context=ctx,
+        data_names = [graph_input for graph_input in self.symbol.list_inputs()
+                      if graph_input not in self.arg_params and graph_input not in self.aux_params]
```

Comment:
should this maybe be a subroutine? Are all of the lines like this the same?

Reply:
This will eventually be an API call to the onnx module like 'get_model_metadata' along with other information like shape of the input and so on. For now I thought this one-liner will be neat and concise. 

Do you suggest I make a method call to make it more readable? Adding parentheses will break the syntax.

Reply:
Or a good comment

Reply:
I will put in a comment.